### ISSUE TITLE
Dark mode support

### ISSUE THREAD

**Issue by sergeyd63** [TylerAHolden/react-schedule-meeting#57]
Hey, 

Pretty obvious from the the title 😁 but does it supports dark mode?

Thanks

**Comment by TylerAHolden** [comment]
Not at the moment i think the light theme is hardcoded in the styles

**Comment by sergeyd63** [comment]
yeah. it mostly works fine, the only issue is with the headers. When we switch to dark the text becomes white on white background 🤔

everything else is fine

**Comment by sergeyd63** [comment]
I selected the text in the first one

<img width="1197" alt="image" src="https://user-images.githubusercontent.com/4738739/205086637-a781d734-9fd3-402c-9f27-42f813fafd72.png">
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/4738739/205086711-7dfa2d72-625d-4b93-8730-9a3f601ee970.png">

thanks 👍

**Comment by TylerAHolden** [comment]
oh use `.rsm-date-title` as a css selector inside the calendar to target those.

**Comment by TylerAHolden** [comment]
When I refactor styles to not require styled-components ill probably add some sort of quick support for dark theming too to make it easy
